### PR TITLE
Fix HA bugzilla link

### PR DESCRIPTION
**Pull request by Martchus**

### CODE REVIEW

File: templates/branding/openSUSE/external_reporting.html.ep [hunk 1]
```diff
@@ -7,16 +7,18 @@
...
     SAP => 'for SAP Applications',
-);
-%>
+    'Server-HA' => 'High Availability Extension',
+); %>
 % my $distri = $distri_to_prod{$job->DISTRI};
 % my $product = '';
 % if ($job->DISTRI eq 'sle') {
+%     my $flavor = $job->FLAVOR;
+%     $flavor =~ s/Server-DVD-HA/Server-HA-DVD/;
```
Comment (aaannz) [inline templates/branding/openSUSE/external_reporting.html.ep]: I don't understand what are you doing here. You initialize `$flavor` from `$job->FLAVOR;` do substitution on `$flavor` but then use `$job->FLAVOR` and `$flavor` is not used anywhere else. Did I miss something?

Comment (Martchus) [inline templates/branding/openSUSE/external_reporting.html.ep]: Don't think you miss something. I'll fix it.

**Comment by aaannz** [comment]
Ok, wait a sec, @okurz come up with this https://github.com/os-autoinst/openQA/pull/1034 for bso vs boo which I liked more.

**Comment by Martchus** [comment]
Then merge his PR first. I will resolve the conflict then.

**Pull request merged**